bc_obps/reporting/service/report_additional_data.py

```diff
@@ -15,11 +15,7 @@ def get_report_report_additional_data(
 
     @staticmethod
     def get_registration_purpose_by_version_id(version_id: int) -> dict:
-        registration_purpose = (
-            ReportVersion.objects.select_related('report', 'report__operation')
-            .get(id=version_id)
-            .report.operation.registration_purpose
-        )
+        registration_purpose = ReportOperation.objects.get(report_version__id=version_id).registration_purpose
```

Comment:
Registration purpose should default to the value in registration. I think we could make the `get_report_report_additional_data` return an in-memory object with that value set this way we wouldn't even have to have that `get_registration_purpose_by_version_id` endpoint